### PR TITLE
fix(mcdu): init a small improvements

### PR DESCRIPTION
**Pull request by MisterChocker**
## Summary of Changes

- [x] Fixed crz fl / tmp spacing
- [x] Fixed From/To text positioning
- [x] Fixed missing ° symbol on crz fl temp
- [x] Fixed small slash on auto crz fl temp

## Screenshots (if necessary)

Before:
image
After:
image
image

## References

image

## Additional context

Discord username (if different from GitHub):

## Testing instructions

- Check From/To text alignment,
- Check CRZ FL /TEMP space,
- Check CRL FL TEMP missing ° on entry boxes are fixed,
- Check CRZ FL TEMP / is small when automatic populated.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js [hunk 1]
```diff
@@ -95,11 +95,11 @@ class CDUInitPage {
                     }
                 );
 
-                cruiseFlTemp = "_____|____[color]amber";
+                cruiseFlTemp = "_____\xa0|____[color]amber";
```
Comment (beheh) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js:98]: This is also missing the degree symbol:

```suggestion
                cruiseFlTemp = "_____\xa0|___°[color]amber";
```

FMS 1A reference, but unlikely this changed:

image

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js [hunk 2]
```diff
@@ -95,11 +95,11 @@ class CDUInitPage {
...
                 //This is done so pilot enters a FL first, rather than using the computed one
                 if (mcdu._cruiseEntered && mcdu._cruiseFlightLevel) {
                     cruiseFlTemp =
-                        "{cyan}FL" + mcdu._cruiseFlightLevel.toFixed(0).padStart(3, "0") + "/" +
+                        "{cyan}FL" + mcdu._cruiseFlightLevel.toFixed(0).padStart(3, "0") + "\xa0/" +
```
Comment (beheh) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js:102]: While you're at it, the slash should be small if TEMP is automatic:

image

image

**Review by beheh — CHANGES_REQUESTED** [review]

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js [hunk 3]
```diff
@@ -95,12 +95,12 @@ class CDUInitPage {
...
 
-                cruiseFlTemp = "_____|____[color]amber";
+                cruiseFlTemp = "_____\xa0|___°[color]amber";
                 //This is done so pilot enters a FL first, rather than using the computed one
                 if (mcdu._cruiseEntered && mcdu._cruiseFlightLevel) {
                     cruiseFlTemp =
-                        "{cyan}FL" + mcdu._cruiseFlightLevel.toFixed(0).padStart(3, "0") + "/" +
-                        (!!mcdu.cruiseTemperature ? mcdu.cruiseTemperature.toFixed(0) + "°" : "{small}" + mcdu.tempCurve.evaluate(mcdu._cruiseFlightLevel).toFixed(0) + "°{end}") +
+                        "{cyan}FL" + mcdu._cruiseFlightLevel.toFixed(0).padStart(3, "0") + "\xa0" +
+                        (!!mcdu.cruiseTemperature ? "/" + mcdu.cruiseTemperature.toFixed(0) + "°" : "{small}/" + mcdu.tempCurve.evaluate(mcdu._cruiseFlightLevel).toFixed(0) + "°{end}") +
```
Comment (beheh) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js:103]: The double negation is redundant as it's evaluated anyway:

```suggestion
                        (mcdu.cruiseTemperature ? "/" + mcdu.cruiseTemperature.toFixed(0) + "°" : "{small}/" + mcdu.tempCurve.evaluate(mcdu._cruiseFlightLevel).toFixed(0) + "°{end}") +
```

**Review by beheh — APPROVED** [review]

**Comment by BritishAvgeek** [comment]
Category: QA Tester
Name: BritishAvgeek#1436
Date of testing: 20/03/2021
Version of the sim: 1.14.5.0
PR Tested: #4033 
Tier of testing: 1
Changes to observe: small changes to INIT A page

Testing technique: Spawned C&D, check on ground
Steps you've made to test the Pull Request:
1. Spawned Cold and Dark and Gatwick, filled out data for a flight in the MCDU
2. Checked the pages mentioned above to text, font or spacing issues - none noticed 
3. Made a few changes to see how spacing is affected, no issues there either
4. End of Test

Flight notes (When done a full flight): N/A

Media: N/A

Issues (If Any): None found
Steps to achieve the issue

Overall Rating: Good
Conclusions of the testing: quick PR test with basic inputs on MCDU from EGKK to EGCC - remained on the ground. No issues found

**Comment by darkfury334** [comment]
Category: QA Trial
Name:DarkFury#4936
Date of testing: Current date 20/03/2021
PR Tested: #4033
Tier of testing: 1
Changes to observe:
 Fixed crz fl / tmp spacing
 Fixed From/To text positioning
 Fixed missing ° symbol on crz fl temp
 Fixed small slash on auto crz fl temp

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:
1. Load in C&D at EGCC
2. Start Batteries / External pover
3. Click INIT page on MDCU
4.Check From/To text alignment,
5.Check CRZ FL /TEMP space,
6.Check CRL FL TEMP missing ° on entry boxes are fixed,
7.Import a flight plan
8.Check CRZ FL TEMP / is small when automatic populated.


Flight notes (When done a full flight):N/A

Media: 
Microsoft Flight Simulator Screenshot 2021 03 20 - 10 50 51 83

Microsoft Flight Simulator Screenshot 2021 03 20 - 11 00 19 72


Issues (If Any): N/A
Overall Rating: Good
Conclusions of the testing: All Good.

**Pull request merged**